content/support/faqs/supported-platforms@es.md

```diff
@@ -0,0 +1,73 @@
+---
+$title@: Supported Platforms, Vendors and Partners
+$order: 3
+$parent: /content/support/faqs.md
+class: who
+
+description@: The Accelerated Mobile Pages (AMP) Project is an open source initiative that makes it easy for publishers to create mobile-friendly content once and have it load instantly everywhere. – Accelerated Mobile Pages Project
+
+cta:
+  title@: Próximo FAQ
+  link_text@: AMP Overview
```

Comment:
I think `AMP Overview` does not must be translated. There is not a word in spanish for "overview".